### PR TITLE
Add team field to Person

### LINKED ISSUE #20: As a team leader, I can add members to my team
... so that I can keep track of the members in my team.

### PR DESCRIPTION
**Pull request by marcuspang**
# Changes
- Add `add_command INDEX` command 
- Update `ReadOnlyAddressBook` to have `getTeam()` method, allowing the `Command` and `CommandParser` classes to retrieve the `Team` object

## Note
Currently, `Team` in `ReadOnlyAddressBook` is not immutable. Not too sure if it should be updated to be immutable in the future. 

Trying to get the tests to work, but `assertCommandSuccess` method doesn't work well when mutating the `Team` object in the test methods

Fixes #20

### CODE REVIEW

**Review by changjy44 — APPROVED** [review]
Good adherence to coding standards. Looks good to merge. Would be nice to see a delete_member command and a visual effect in the UI upon adding a member to the team.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -46,20 +50,20 @@ TruthTable is a **desktop app for managing software engineering teams, optimized
...
 Add a new team member to the user’s team. Will throw an error if the member already exists in the team.
 
 Format: 
-* `add member “MEMBER_NAME”`
+* `add_member “INDEX”`
 
 Examples:
-* `add member egg`
-* `add member “potato egg”`
+* `add_member 1`
```
[the remark this answers is not in the document]
Comment (marcuspang) [reply docs/UserGuide.md:61]: The members added should be a `Person` model, which was why I felt it was better to use an index like the other commands.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -46,20 +50,20 @@ TruthTable is a **desktop app for managing software engineering teams, optimized
...
 
 
-### Adding a new member to the team `add member`
+### Adding a new member to the team `add_member`
 
 Add a new team member to the user’s team. Will throw an error if the member already exists in the team.
 
 Format: 
-* `add member “MEMBER_NAME”`
+* `add_member “INDEX”`
```
[the remark this answers is not in the document]
Comment (marcuspang) [reply docs/UserGuide.md:58]: That makes sense, I'll remove it

**Comment by marcuspang** [comment]
> Good adherence to coding standards. Looks good to merge. Would be nice to see a delete_member command and a visual effect in the UI upon adding a member to the team.

The delete_member command will be done in a separate PR. I think I won't add a visual effect to the so early, perhaps only when the Team model is implemented properly

**Pull request merged**